skxray/cdi.py

```diff
@@ -96,6 +98,27 @@ def gauss(dims, sigma):
     return y / np.sum(y)
 
 
+def _fft_helper(x, option='fft'):
```

Comment:
I would suggest two functions `_fft_helper` and `_ifft_helper`.  Switching on a string like this is not a great pattern.  If you really want to be able to switch based on a string key, I would implement this as a dictionary look up of lambda functions.  This will fail in really surprising ways if option is not 'fft' or 'ifft'.